esd_services_api_client/beast/v3/_connector.py

```diff
@@ -254,6 +254,16 @@ def get_request_lifecycle_stage(self, request_id: str) -> Optional[str]:
 
         return response.json()["lifeCycleStage"]
 
+    def get_request_runtime_info(self, request_id: str) -> Optional[Dict]:
```

Comment:
```suggestion
    def get_request_runtime_info(self, request_id: str) -> Optional[dict]:
```
Let's start using Py3.11 default syntax :)